tests/queries/0_stateless/02935_http_content_type_setter_for_json.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# for frmt in JSONStrings JSON JSONEachRow JSONColumns JSONColumnsWithMetadata JSONCompact JSONCompactStrings JSONCompactColumns PrettyJSONEachRow JSONEachRowWithProgress JSONStringsEachRow JSONStringsEachRowWithProgress JSONCompactEachRow JSONCompactEachRowWithNames JSONCompactEachRowWithNamesAndTypes JSONCompactStringsEachRow JSONCompactStringsEachRowWithNames JSONCompactStringsEachRowWithNamesAndTypes JSONObjectEachRow
+for frmt in JSONStrings JSON JSONEachRow JSONColumns JSONCompact JSONObjectEachRow
+do
+  echo $frmt
+  url="${CLICKHOUSE_URL}/&output_format_json_content_type_header=custom-type&query=select+sleepEachRow(0.01)from+numbers(10)+FORMAT+${frmt}"
```

Comment:
Why do we need `sleepEachRow` here?

Reply:
no need. It's cloned from 02935_http_content_type_with_progress.

Reply:
Let's remove, it slows down the test